### PR TITLE
doc: Remove local provider references

### PR DESCRIPTION
**Pull request by Moneyba**
## Description
Remove all local provider references

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2410,7 +2408,6 @@ Currently, the Booster framework provides a fully working provider package:
 
 Other providers packages are currently under development. Some of the features might be missing:
 
--  **framework-provider-local-\***. The Booster framework local provider combines in-memory databases with a GraphQL API served through a Node.js Express Server. The local runtime is a convenient and fast way to deploy and test your code in a local development environment. From the API and semantic perspectives, there are no differences from using a real cloud provider, it just runs locally!
 - **framework-provider-kubernetes-\***
 - **framework-provider-azure-\***
```
Comment (davidverdu) [inline docs/README.md:2413]: Can you explain that Azure and Kubernetes are already implemented and not currently under development?

Comment (Moneyba) [reply docs/README.md:2413]: But they are not fully working, right? 
So, should I say something like they're implemented but they're not fully working for now?

Comment (davidverdu) [reply docs/README.md]: Correct. We should explain we have experimental support on these providers

File: docs/README.md [hunk 2]
```diff
@@ -2441,7 +2438,7 @@ The following is the list of the fields you can configure:
 
 - **provider:** This field contains the provider library instance that Booster will use when deploying or running your application.
 
-> **Note:** So far, there is only one provider fully supported in Booster yet, @boostercloud/framework-provider-aws, and it is probably the one you have already set if you used the generator to create your project. The team is currently working on providers for local development, Azure, and Kubernetes._
+> **Note:** So far, there is only one provider fully supported in Booster yet, @boostercloud/framework-provider-aws, and it is probably the one you have already set if you used the generator to create your project. The team is currently working on providers for Azure and Kubernetes.
```
Comment (davidverdu) [inline docs/README.md:2441]: Azure and Kubernetes have experimental support and the team is currently improving them

**Review by davidverdu — APPROVED** [review]
LGTM. I left some notes in comments

**Review by AdrianLorenzoDev — APPROVED** [review]
LGTM!

**Pull request merged**